### DISCUSSION TITLE
JS file is not loading with pimcore custom bundle

### DISCUSSION THREAD

**Discussion by shubhangi101092** [pimcore/pimcore#16396] [Q&A / Support]
Hi ,

I have generated new custom bundle and its working as making new route in controller which is working fine but when i do some change in startup.js inside pimcoreready function which is coming by default its not working , i cleared cache and install the assets and checked with different browser but its not working.Pimcore version is 10.5

**Comment by neha04** [comment]
@shubhangi101092 Do you have this fn in the {bundlename}.php file ?

    public function getJsPaths() {
        return [
            '/bundles/{bundlename}/js/pimcore/startup.js'
        ];
    }

**Reply by shubhangi101092** [reply]
Yes @neha04 that is coming default when installing bundle
